src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -25,6 +25,14 @@ internal static partial class CpuMathUtils
         public static int GetVectorAlignment()
             => Avx.IsSupported ? Vector256Alignment : (Sse.IsSupported ? Vector128Alignment : FloatAlignment);
 
+        /// <summary>
+        /// Multiplies a matrix times a source.
+        /// </summary>
+        /// <param name="transpose"><see langword="true"/> to transpose the matrix; otherwise <see langword="false"/>.</param>
```

Comment:
Overall this is exactly the kind of comment I would like to see for a parameter. There are two changes I would suggest:

1. Using a parameter reference so it's clear which matrix will be transposed (this is my bad for not being clearer earlier)
2. Add a comma to match the form seen throughout docs.microsoft.com

```suggestion
        /// <param name="transpose"><see langword="true"/> to transpose <paramref name="matrix"/>; otherwise, <see langword="false"/>.</param>
```

Reply:
Since this PR is old and a net improvement I suggest merging as is and this can be done subsequently